Userland/reboot.cpp

```diff
@@ -0,0 +1,13 @@
+#include <Kernel/Syscall.h>
+
+
+int main(int argc, char** argv)
```

Comment:
Instead of `(void)`'ing these arguments below, you can just omit their names. This will prevent the compiler from warning about them being unused.
```suggestion
int main(int, char**)
```